optaplanner-examples/src/main/resources/org/optaplanner/examples/vehiclerouting/vehicleRoutingSolverConfig.xml

```diff
@@ -71,7 +71,17 @@
           </nearbySelection>
         </secondarySubListSelector>
       </subListSwapMoveSelector>
-      <kOptListMoveSelector/>
+      <kOptListMoveSelector>
```

Comment:
Does the default config still work?
Is the old way of configuring K-Opt (which didn't use value selectors) still supported?

Reply:
Yes (all config parameters are optional; not specifying them result in a default value selector being created)